data/src/main/java/com/microsoft/azure/kusto/data/exceptions/TimespanParseException.java

```diff
@@ -0,0 +1,8 @@
+package com.microsoft.azure.kusto.data.exceptions;
+
+public class TimespanParseException extends RuntimeException {
```

Comment:
I personally don't see the benefit of creating a new class of exceptions for each exception scenario.
I see exception classes as being categories, and their `String message` as providing more specific details.

Java already has a https://docs.oracle.com/javase/8/docs/api/java/text/ParseException.html?is-external=true for the general case, and if you end up using Duration (or anything from the Java Time library), has https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeParseException.html